models/classes/Lists/DataAccess/Repository/RdfValueCollectionRepository.php

```diff
@@ -381,4 +392,26 @@ private function getPersistence(): SqlPersistence
         /** @noinspection PhpIncompatibleReturnTypeInspection */
         return $this->persistenceManager->getPersistenceById($this->persistenceId);
     }
+
+    private function extractLabel(ValueCollectionSearchRequest $searchRequest, iterable $labels, $subject): ?string
```

Comment:
```suggestion
    private function extractLabel(ValueCollectionSearchRequest $searchRequest, iterable $labels, $subject): string
```
I can't see circumstances for this method to return `null`